.github/workflows/main.yml

```
@@ -0,0 +1,114 @@
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch on PR
+  push:
+    branches: '**'
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        ocaml-compiler:
+          - 4.13.x
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Use OCaml ${{ matrix.ocaml-compiler }}
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+
+      - run: opam install . --deps-only --with-test
+
+      # Runs a set of commands using the runners shell
+      - name: 5.0.0+stable
+        run: |
+          sudo apt-get update && sudo apt-get -y install wget pkg-config libgmp-dev m4 libdw-dev jq python3-pip autoconf jo
+          pip3 install intervaltree
+          eval $(opam env)
+          opam update
+          opam install dune.2.9.0
+          eval $(opam env)
+          export ITER=1
+          export OPAM_DISABLE_SANDBOXING=true
+          sed 's/--cpu-list 5/--cpu-list 1/g' run_config_filtered.json
+          TAG='"run_in_ci"' make run_config_filtered.json
+          USE_SYS_DUNE_HACK=1 RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/5.0.0+stable.bench
```

```suggestion
          TAG='"run_in_ci"' make run_config_filtered.json
          sed -i 's/--cpu-list 5/--cpu-list 1/g' run_config_filtered.json
          USE_SYS_DUNE_HACK=1 RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/5.0.0+stable.bench
```

The `sed` command should be called after the `run_config_filtered.json` file is generated. Also, you need to add a `-i` flag to the sed command to edit the file in-place.

---

@Sudha247 Thanks a lot for this suggestion, I'll commit them ASAP.